prombench/manifests/prombench/benchmark/7_prometheus_remote_write.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prometheus-sink
+  namespace: prombench-{{ .PR_NUMBER }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sink
+  template:
+    metadata:
+      namespace: prombench-{{ .PR_NUMBER }}
+      labels:
+        app: sink
+    spec:
+      containers:
+      - name: prom-sink
+        image: quay.io/bwplotka/sink:latest
+        imagePullPolicy: Always
+        ports:
+        - name: sink-port
+          containerPort: 9011
+      nodeSelector:
+        node-name: nodes-{{ .PR_NUMBER }}
+        isolation: none
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus-remote-write
```

Comment:
I think it might be easier if this were named the same as the Deployment.
That's the pattern used for `fake-webserver`, `prometheus-loadgen-querier`.

Reply:
okay.